package.json

```diff
@@ -17,7 +17,9 @@
     "dependencies": {
         "cli-progress": "^3.9.0",
         "commander": "^7.2.0",
+        "fdir": "^5.1.0",
```

Comment:
>fdir [](http://example.com/codeflow?start=9&length=4)

Looks like we should add a dep on `picomatch`:
[https://github.com/thecodrr/fdir/blob/master/documentation.md#globstring](https://github.com/thecodrr/fdir/blob/master/documentation.md#globstring)

Reply:
Good catch. I think I had glob installed alongside this prior which is why it was working locally.